Heading/Heading.js

```diff
@@ -108,11 +111,19 @@ const HeadingBase = kind({
  *
  * @hoc
  * @memberof agate/Heading
+ * @mixes agate/Marquee.MarqueeDecorator
  * @mixes agate/Skinnable.Skinnable
  * @public
  */
 const HeadingDecorator = compose(
+	setPropTypes({
+		marqueeOn: PropTypes.oneOf(['hover', 'render'])
+	}),
+	defaultProps({
+		marqueeOn: 'render'
+	}),
```

Comment:
My understanding for these lines is..

the `marqueeOn` prop in `packages/ui/Marquee/MarqueeDecorator.js` has `focus` as a default value and could have `hover`, `render`, and `focus`. But the `Header` can't be focused. So the default should be another value except `focus` and the possible values would be `render` and `hover`.

This PR provides the way to select one value between `hover` and `render` nicely instead of choosing one component side. But we need to make a decision what the UX is correct aligning with next Agate UX.

In addition, there is a new way to define defaultProp and propTypes from recompose witth `defaultProps` and `setPropTypes` in our components. I think this approach is quite useful to define HoC.

I like these ways. So LGTM. But I want to hear merger's opinion.

P.S: No doc parse error

Reply:
I agree that having the choice of "hover" and "render" is appropriate for `heading`.
And I believe this is suitable for the next Agate UX as well.
I've confirmed that the way to define props exactly same as Sandstone. I like to maintain this identity. So LGTM too.
@0x64  @MikyungKim Do you have any other opinions?